ee/desktop/menu/dark_mode_windows.go

```diff
@@ -0,0 +1,27 @@
+//go:build windows
+// +build windows
+
+package menu
+
+import (
+	"golang.org/x/sys/windows/registry"
+	"log"
+)
+
+const (
+	regKey  = `Software\Microsoft\Windows\CurrentVersion\Themes\Personalize` // in HKCU
+	regName = `SystemUsesLightTheme`                                         // <- For taskbar & tray. Use AppsUseLightTheme for apps
+)
+
+func isDarkMode() bool {
+	k, err := registry.OpenKey(registry.CURRENT_USER, regKey, registry.QUERY_VALUE)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
log.Fatal() will cause os.Exit(1). Are we sure we want this here? I guess if we couldn't read the registry we would end up in a crash loop.

If this was intentional, can you put some comments explaining why?

Reply:
oops no this was not intentional, I'll remove this

Reply:
fixed